src/Pages/NotesPage/NoteComponents/NewNotes/NewNotes.jsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from "react";
+import { useNoteContext } from "../../../../Context/Notes.context";
+import { EditModel } from "../EditModel/EditModel";
+import "./NewNotes.css";
+
+export const NewNotes = () => {
+  const { notes, removeNote, editNote, pinHandler, isEditMode, setIsEditMode } =
+    useNoteContext();
+
+  return (
+    <>
+      <div className="new-note-container">
+        {notes.filter((n) => n.pinned).length === 0 ? null : (
+          <h1 className="pin-text">Pinned</h1>
+        )}
+        <div className="pinned">
+          {notes
+            .filter((n) => n.pinned)
+            .map((note) => {
+              return (
+                <div
+                  key={note._id}
+                  className="newnote-input-container card-shadow"
+                >
+                  <div className="input-text-section-container">
+                    <div className="input-text-section">
+                      <h2>{note.title}</h2>
+                      <h3>{note.description}</h3>
+                    </div>
+                    <div>
+                      <span
+                        onClick={() => pinHandler(note)}
+                        className="material-icons pin-icon active"
+                      >
+                        push_pin
+                      </span>
+                    </div>
+                  </div>
+                  <div className="edit-section-container">
+                    <div className="edit-section">
+                      <h2>{note.typeOfNote}</h2>
+                    </div>
+                    <div className="note-icon-container">
+                      <span
+                        onClick={() => setIsEditMode({ state: true, note: note })}
+                        className="material-icons pin-icon"
+                      >
+                        edit
+                      </span>
+                      <span className="material-icons pin-icon">archive</span>
+                      <span
+                        onClick={() => removeNote(note._id)}
+                        className="material-icons pin-icon"
+                      >
+                        delete
+                      </span>
+                    </div>
+                  </div>
+                </div>
+              );
+            })}
+        </div>
```

Comment:
You can create a component for this code chunk. It will decrease the file clutter and you can use it in other places.

Reply:
sure will do that